lib/Backends/OpenCL/OpenCL.h

```diff
@@ -246,6 +246,8 @@ class OCLBackend final : public BackendUsingGlowIR {
     // The group convolution is supported in OpenCL slow convolution kernel.
     if (N->getKind() == Kinded::Kind::ConvolutionNodeKind)
       return false;
+    if (N->getKind() == Kinded::Kind::ReluNodeKind)
```

Comment:
Please add a comment explaining why you don't want to lower ReLUs.

Reply:
And BTW, we usually use {} even for a single line if/else/for/while bodies. I see that the line above was not following this style ;-) Please change it as well.